### PR TITLE
refactor(jimp): update jimp to fix minimist vulnerability

### LINKED ISSUE #121: Use latest jimp to fix minimist security vulnerability
Repro:
```
PS C:\dev\temp> npm init -y
Wrote to C:\dev\temp\package.json:
```
```
PS C:\dev\temp> npm i webpack-pwa-manifest
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
...
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN temp@1.0.0 No description
npm WARN temp@1.0.0 No repository field.

+ webpack-pwa-manifest@4.2.0
added 69 packages from 98 contributors and audited 163 packages in 6.198s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```
```
PS C:\dev\temp> npm audit

                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance


  Low             Prototype Pollution

  Package         minimist

  Patched in      >=0.2.1 <1.0.0 || >=1.2.3

  Dependency of   webpack-pwa-manifest

  Path            webpack-pwa-manifest > jimp > @jimp/custom > @jimp/core >
                  mkdirp > minimist

  More info       https://npmjs.com/advisories/1179

found 1 low severity vulnerability in 163 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```
```
PS C:\dev\temp> npm audit fix
npm WARN temp@1.0.0 No description
npm WARN temp@1.0.0 No repository field.

up to date in 0.349s
fixed 0 of 1 vulnerability in 163 scanned packages
  1 vulnerability required manual review and could not be updated
```

The latest jimp should fix it as per https://github.com/oliver-moran/jimp/pull/857

### PR DESCRIPTION
**Pull request by Den-dp**
closes #121

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   "license": "MIT",
   "dependencies": {
     "css-color-names": "^1.0.0",
-    "jimp": "^0.6.0",
+    "jimp": "^0.9.6",
```
Comment (thasmo) [inline package.json:34]: There's even `v0.10.0` now. 👍

Comment (Den-dp) [reply package.json]: bumped

**Comment by hc-12** [comment]
Can we merge this PR?

**Comment by rajatb-git** [comment]
Been waiting for this fix... when will this be merged?

**Comment by Den-dp** [comment]
It would be nice if @arthurbergmz or @doug-wade looked at this PR.

But I would say that we are blocked with #119 🤷‍♂️

**Comment by bcanseco** [comment]
Checked out @Den-dp's fork - all tests passed. Used it in one of my projects and images are processed as expected. Bumping for a wishful merge 🙏

**Comment by Den-dp** [comment]
I decided to go with publishing this fix from my own fork. For that I want to configure npm publishing from a `master` branch (b/c this is how semantic-release package works), but unfortunately this PR was provided via `master` branch too 🤦.
So have to close it on order to add some configuration for publishing.

**Comment by Den-dp** [comment]
So I released a fix from my fork https://github.com/Den-dp/webpack-pwa-manifest/releases/tag/v4.2.1 🙌 

You can install it via:
```
npm i webpack-pwa-manifest-contrib@4.2.1 -D
```
Please notice the new name with `-contrib` at the end, and make sure to change `require('webpack-pwa-manifest-contrib')` in `webpack.config.js`.

**Comment by Stanzilla** [comment]
@Den-dp nice! are you planning to integrate more of the open pull requests from here? maybe try posting in https://github.com/arthurbergmz/webpack-pwa-manifest/issues/119 as well so @arthurbergmz could give you permissions on here?

**Pull request closed without merging**